definitions/tools/add_strelka_gt.cwl

```diff
@@ -5,11 +5,99 @@ class: CommandLineTool
 label: "add GT tags"
 requirements:
     - class: DockerRequirement
-      dockerPull: "mgibio/cle:v1.3.1"
+      dockerPull: "ubuntu:xenial"
```

Comment:
Is it time to use `bionic` instead? :smile:

Reply:
Good point, we're not even that far off from `focal` now.. updating